final_prep/annotations.rb

```diff
@@ -5,39 +5,68 @@
 # Build a Bear
 
 def build_a_bear(name, age, fur, clothes, special_power)
+  #defines new method called 'build a bear' with several parameters
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #defines string with name variable
   demographics = [name, age]
+  #defines demographics with array
   power_saying = "Did you know that I can #{special_power}?"
+  #defines string with special_power variable
+
   built_bear = {
+    #this creates a hash which stores info about built_bear.
     'basic_info' => demographics,
+    #this assigns 'basic info' to 'demographics'
```

Comment:
You can think about showing where those values come from- are they declared as a variable? Did you get them from the params? Sometimes, it takes some dete3ctive work to find where a value comes from.